packages/prepo-stores/src/utils/CustomProviders.ts

```diff
@@ -84,6 +84,11 @@ export class FallthroughProvider extends ethers.providers.BaseProvider {
       // alternatively, consider having an rpcFailed flag which triggers
       // a different UI when all RPCs are down (e.g. prompt user to add custom RPC)
       this.options.onError?.(e as Error)
+      if (isProduction) {
+        throw e
+      } else {
+        throw Error('No RPCs available at the moment.')
+      }
```

Comment:
well actually these lines can be ignored because there's already the line to throw no rpcs available error
```suggestion
      }
```